File: Pregunta2_Secuenciales/Pregunta2.cs
using System;
using System.Globalization;

class Pregunta2 {
    static void Main() {
        CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
        Console.Write("Ingrese el número de productos N: ");
        int N = int.Parse(Console.ReadLine());

        double subtotal = 0;

        for (int i = 1; i <= N; i++) {
            Console.WriteLine($"Producto {i}:");
            Console.Write("  Precio: ");
            double precio = double.Parse(Console.ReadLine());
            Console.Write("  Cantidad: ");
            int cantidad = int.Parse(Console.ReadLine());
            subtotal += precio * cantidad;
        }

        double igv = subtotal * 0.18;
        double total = subtotal + igv;

        Console.WriteLine($"Subtotal: {subtotal:F2}");
        Console.WriteLine($"IGV: {igv:F2}");
        Console.WriteLine($"Total a pagar: {total:F2}");
    }
}

File: Pregunta2_Secuenciales/pregunta2.py
n = int(input("Ingrese el número de productos N: "))
subtotal = 0.0

for i in range(1, n+1):
    print(f"Producto {i}:")
    precio = float(input("  Precio: "))
    cantidad = int(input("  Cantidad: "))
    subtotal += precio * cantidad

igv = subtotal * 0.18
total = subtotal + igv

print(f"Subtotal: {subtotal:.2f}")
print(f"IGV: {igv:.2f}")
print(f"Total a pagar: {total:.2f}")
